tools/github_webhook_processor.php

```diff
@@ -105,10 +105,96 @@
 		print_r($payload); # For debug only. Can be found in GitHub hook log.
 		die();
 }
+
+//rip bs-12
+function tag_pr($payload, $opened) {
+	global $apiKey;
+
+	//We need to reget the pull_request part of the payload to actually see the mergeable field populated
+	//http://stackoverflow.com/questions/30619549/why-does-github-api-return-an-unknown-mergeable-state-in-a-pull-request
+	$scontext = array('http' => array(
+		'method'	=> 'GET',
+		'header'	=>
+			"Content-type: application/json\r\n".
+			'Authorization: token ' . $apiKey,
+		'ignore_errors' => true,
+		'user_agent' 	=> 'tgstation13.org-Github-Automation-Tools'
+	));
+
+	$url = $payload['pull_request']['url'];
+	$payload['pull_request'] = json_decode(file_get_contents($url, false, stream_context_create($scontext)), true);
+	if($payload['pull_request']['mergeable'] == null) {
+		//STILL not ready. Give it a bit, then try one more time
+		sleep(10);
```

Comment:
The merge conflict thing is nice, and it being removed automatically is honestly essential to have.